src/main/java/Duke.java

```diff
@@ -1,10 +1,129 @@
+import java.util.Scanner;
+
 public class Duke {
+    private static final String LOGO =
+                    " _______  __   __  ______   _______  _______  _______  __   __  _______\n" +
+                    "|       ||  | |  ||      | |       ||       ||   _   ||  |_|  ||   _   |\n" +
+                    "|    ___||  | |  ||  _    ||    ___||_     _||  |_|  ||       ||  |_|  |\n" +
+                    "|   | __ |  |_|  || | |   ||   |___   |   |  |       ||       ||       |\n" +
+                    "|   ||  ||       || |_|   ||    ___|  |   |  |       ||       ||       |\n" +
+                    "|   |_| ||       ||       ||   |___   |   |  |   _   || ||_|| ||   _   |\n" +
+                    "|_______||_______||______| |_______|  |___|  |__| |__||_|   |_||__| |__|\n";
+    private static final String SEPARATOR = "zzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzz\n";
+    private static Task[] userInputs = new Task[100];
+    private static int userInputCount = 0;
+
+    /**
+     * Initiate the program and print the various messages
+     */
+    public static void initiateDuke() {
+        System.out.println(SEPARATOR + "Hi... from GUDETAMA... so sleepy\n" + LOGO);
+        System.out.println("Give me five more minutes..... What can I do for you?\n" + SEPARATOR);
+    }
+
+    /**
+     * Takes in user inputs and process the inputs accordingly
+     */
+    public static void processInput() {
+        String line;
+        Scanner in = new Scanner(System.in);
+        line = in.nextLine();
+
+        while (!line.equals("bye")) {
+            if (line.equals("list")) {
+                listTasks();
+            } else if (line.startsWith("done")) {
+                markDone(line);
+            } else if (line.startsWith("todo")) {
+                addTodo(line);
+            } else if (line.startsWith("deadline")) {
+                addDeadline(line);
+            } else if (line.startsWith("event")) {
+                addEvent(line);
+            } else {
+                handleInvalid();
+            }
+            line = in.nextLine();
+        }
+        endDuke();
+    }
+
+    /**
+     * Prints the ending messages before ending the program
+     */
+    public static void endDuke() {
```

Comment:
This function is not doing what the name is suggesting, as it is not terminating the application